controllers/etcd_controller.go

```diff
@@ -209,12 +209,12 @@ func (r *EtcdReconciler) reconcile(ctx context.Context, etcd *druidv1alpha1.Etcd
 	}
 	if err := r.addFinalizersToDependantSecrets(ctx, etcd); err != nil {
 		if err := r.updateEtcdErrorStatus(etcd, nil, err); err != nil {
+			logger.Infof("Reconciling etcd New")
```

Comment:
Shouldn't this be logged at the beginning of the reconciliation? Or if here shouldn't we log that we are requeueing for reconciliation?

Reply:
Remanent of some testing logs